shoes-core/spec/shoes/star_spec.rb

```diff
@@ -202,4 +202,16 @@
       expect(subject.redraw_height).to eq(108)
     end
   end
+
+  describe "center_point" do
+    subject { Shoes::Star.new(app, parent, 100, 100, 5, 50, 30, center: false, strokewidth: 0) }
```

Comment:
One other test that might be worth adding is explicitly setting some of the dimensions to `nil` values. This mimics some scenarios during app startup where we might not be fully positioned yet. I typically make derived values like your `center_point` just do something sensible (`nil` or `0` return) if their underlying values are missing.

Reply:
Ok, cool!  Thanks.